apiary.apib

```diff
@@ -2039,6 +2039,9 @@ Get details for a single invoice.
                     + payment (object)
                         + Include Money
             + payment_reference: `+++084/2613/66074+++` (string)
+            + digital_invoicing (object, nullable)
+                + (object)
```

Comment:
Q: the digital_invoicing object doesn't contain another object, right?
I think we can drop this line.

Reply:
You are right! Thanks for spotting it 👌